compiler/rustc_mir_dataflow/src/move_paths/builder.rs

```diff
@@ -207,13 +207,12 @@ impl<'b, 'a, 'tcx> Gatherer<'b, 'a, 'tcx> {
     }
 }
 
+pub type MoveDat<'tcx> = (FxHashMap<Local, Place<'tcx>>, MoveData<'tcx>);
```

Comment:
I meant the entire return type including the `Result`.

Also... independently from that. `MoveData` is returned both in the success and the failure case, maybe we should be returning `(MoveData<'tcx>, Result<.., Vec<(Place<'tcx>, MoveError<'tcx>)>>)`

Reply:
I could do that but I think that would result in a lot more changes because some places require both move_data and move_errors to report the error, so changing those would result in changes that are. completely unrelated to this PR.